ResearchKit/Common/ORKHTMLPDFWriter.m

```diff
@@ -89,8 +89,8 @@ @interface ORKHTMLPDFWriter () <UIWebViewDelegate>
 @property (nonatomic) CGSize pageSize;
 @property (nonatomic) UIEdgeInsets pageMargins;
 @property (nonatomic, strong) UIWebView *webView;
-@property (nonatomic, strong) NSData *data;
-@property (nonatomic, strong) NSError *error;
+@property (nonatomic, copy) NSData *data;
+@property (nonatomic, copy) NSError *error;
```

Comment:
NSError - ok.
NSData - less sure. This is an internal property of this method, which constructs mutable data and then sets it as the value of the property. Copying the mutable data is unnecessary, since we know we will never mutate it further after setting it.

Admittedly we never call the setter, since the ivar is set directly, but it's still not obvious this shouldn't just be strong.